packages/components/src/_components/animated-image/animated-image.stories.ts

```diff
@@ -6,7 +6,7 @@ const { argTypes, args, parameters } = storybookDefaults('sd-animated-image');
 const { generateTemplate } = storybookTemplate('sd-animated-image');
 
 export default {
-  title: 'Components/sd-animated-image',
+  title: 'SDS Components/sd-animated-image',
```

Comment:
I believe the problem is that the Site Title is generated via JS here, so it works in the browser – but is not picked up by Open Graph Stuff. I would therefore strongly suggest to remove the SDS here everywhere, as it just bloats the docs and we furthermore talk about "Solid Components" and not "Solid Design System Components".

<img width="244" alt="CleanShot 2023-08-15 at 07 07 02@2x" src="https://github.com/solid-design-system/solid/assets/26542182/e4d2e725-3637-4c97-a0c3-7b75622c3a2c">

Reply:
As the page title is dynamic in Storybook and there have been lots of discussions if and how to solve this (and most of them seemed to be very hacky), I just tried to improve it a bit for eg. the title shown in tabs in the browser.

Currently it shows the "chapter" and then the source tree of the file the user is in: 
![image](https://github.com/solid-design-system/solid/assets/48211526/69232f89-6782-4060-b40b-8a2e9ff76ed1)

By changing it to SDS, it at least shows that the page is part of the SDS:
![image](https://github.com/solid-design-system/solid/assets/48211526/b7e79c6f-9adf-4c05-b12c-2ef77a88cf80)

![image](https://github.com/solid-design-system/solid/assets/48211526/754be0d8-4c5a-4dce-bc9a-7e6c77cf8e45)

I dont see where it bloats the docs here as we just added a prefix to the name but I see the point to change it to something like "Solid Docs" and "Solid Components".

Reply:
As you want only to change what is shown in the browser I found a possible solution:

![Uploading CleanShot 2023-08-18 at 17.36.32.png…]()

Replace every occurance of the following in all files `chunks*.js` in the folder sb-manager.

```
\u22C5 Storybook
```

We could alter the word `Storybook` to `Solid Design System (Storybook)` instead of appending SDS everywhere.